spec/features/support/providers/creating_a_new_provider_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Creating a provider' do
+  let(:user) { create(:user, :admin) }
```

Comment:
I thought we were just sticking to instance variables for these feature specs? I know some original ones used lets but i remember at some point we decided to just stick with `@` so should probs be consistent

Reply:
Done